renderer/renderer.go

```diff
@@ -40,6 +42,25 @@ func New() Renderer {
 	}
 }
 
+// Healthcheck calls the healthcheck endpoint on the renderer and returns any errors
+func (r *renderer) Healthcheck() error {
```

Comment:
Possibly the healthcheck method should return name and error, removes the need for the name function then?